vscode-trace-extension/src/extension.ts

```diff
@@ -6,8 +6,12 @@ import { TraceExplorerAvailableViewsProvider } from './trace-explorer/available-
 import { TraceExplorerOpenedTracesViewProvider } from './trace-explorer/opened-traces/trace-explorer-opened-traces-webview-provider';
 import { fileHandler, openOverviewHandler, resetZoomHandler } from './trace-explorer/trace-tree';
 import { updateTspClient } from './utils/tspClient';
+import { TraceExtensionLogger } from './utils/trace-extension-logger';
```

Comment:
Can we split this PR into at least two commits? Say, one for this logging added, and another for the progress bar addition. If there are refactorings not pertaining to either one of these two main topics, then that should be at least yet another commit.

Doing so enables reviewability, and revertabiliy (should either one of the main changes be deemed problematic after merges). If they all sit in the same commit, then one has to revert everything in case of only some revertworthy bug(s).

Now, if a commit depends on (or builds on top of) another, then it can simply be based on the latter, as usual with git commits indeed.

Reply:
Done! https://github.com/eclipse-cdt-cloud/vscode-trace-extension/pull/129

Once the logging commit is merged, i will update this PR. Thanks!